tests/tests.toml

```
@@ -4271,7 +4271,8 @@ run = true
 
 [[test]]
 filename = "multi_error1.f90"
-ast = true
+ast = true # TODO: remove this and only use continue compilation
+continue_compilation = true
```

once the above comments are addressed ( in subsequent PRs ) this will be converted to simply

```suggestion
continue_compilation = true
```

This will generate valid AST -> ASR -> LLVM -> binary -> execute it. RIght now AST is not valid.